decidim-meetings/app/views/decidim/meetings/meetings/index.js.erb

```diff
@@ -1,6 +1,10 @@
 var $meetings = $('#meetings');
 var $meetingsCount = $('#meetings-count');
 
+// make sure that calendar modal will use the updated filter values
+var $calendar_url = $('#calendarShare');
```

Comment:
Could you use the same naming convention, i.e. camel case (`$calendarUrl`) as with the other variables?